src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,33 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.14.0
+
+[DATE]
+
+This version is compatible with Kubernetes versions 1.24 to 1.27.
+
+### Deprecatation Notice
+
+- As of this versions deploying helm charts through the Okteto UI has been deprecated.
+- Private Registry credentials defined in helm has been deprecated. Private Registry Credentials are now supported natively by Okteto and credentials defined in helm will be automatically migrated. More info [here](ADD_LINK)
```

Comment:
@maroshii @ifbyol my understanding is that there is a migration guide in this case. Something like:
- upgrade
- remove `privateRegistry`
- upgrade again
And we could link to the CRDs doc too

Reply:
Yes, but the PR is not merged yet: https://github.com/okteto/docs/pull/509. That's why there is a placeholder for the link. As soon as we merge the other, we will update this with the link to that guide

Reply:
Link for the migration guide: https://www.okteto.com/docs/1.14/self-hosted/install/upgrade/#upgrading-to-okteto-114x.

We have to remove the `1.14`. from the docs so we always way to latest version